packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -1104,8 +1104,7 @@ Widget _buildCupertinoDialogTransitions(BuildContext context, Animation<double>
 /// barrier behavior (by default, the dialog is not dismissible with a tap on
 /// the barrier).
 ///
-/// This function takes a `builder` which typically builds a [CupertinoDialog]
-/// or [CupertinoAlertDialog] widget. Content below the dialog is dimmed with a
+/// This function takes a `builder` which typically builds a [CupertinoAlertDialog] widget. Content below the dialog is dimmed with a
```

Comment:
Keep the general width per line to be consistent with the rest of the file.

Reply:
You are right, sorry.